public/status/index.html

```diff
@@ -1,177 +1,187 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="x-ua-compatible" content="ie=edge" />
-    <meta name="keywords" content="fanfiction, transformative works, otw, fair use, archive" />
-    <meta name="language" content="en-US" />
-    <meta name="subject" content="fandom" />
-    <meta name="description" content="An Archive of Our Own, a project of the
+
+<head>
```

Comment:
The original indenting in this file definitely needed some help, but I don't think these changes are quite right. `<head>` and `<body>` should be indented 2 spaces deeper than `<html>`, like in the other files. (And everything inside them should be indented 2 more spaces for each level of nesting.)

Reply:
Thank you! In case anyone else encounters a similar issue (and somehow stumbles across this), VSCode doesn't indent <head> and <body> tags by default, there's a setting under 'HTML › Format: Indent Inner HTML' which I needed to change. I think it is all fixed now.